### PR TITLE
Add figma link in issue template

### LINKED ISSUE #15: Issue Template Updates for Figma link
### Current Behavior
The project's current issue templates are missing an open invitation link where new contributors can join the community's Figma team and view user interface designs and other UX projects.

### Desired Situation
Each template that has a reference to Figma in its resources section should an invite link added.

### Implementation 
```
- 🎨 Wireframes and Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in  [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
```

**Acceptance Tests**
All references to Figma include the "open invite" link.

---
**[Contributor Guide](https://docs.meshery.io/project/contributing)**

### PR DESCRIPTION
**Pull request by beelchester**
**Description**
Added figma link in issue template as per the instructions
This PR fixes #15 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 1]
```diff
@@ -14,3 +14,4 @@ assignees: ''
 - [Meshery documentation site](https://docs.meshery.io/)
 - [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
 - [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
+- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
```
Comment (thisiskaransgit) [inline .github/ISSUE_TEMPLATE/documentation.md]:
```suggestion
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
```

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]
@sahilyeole, we needed the layer5 design file here in place of Meshery UI, please update the PR

File: .github/ISSUE_TEMPLATE/documentation.md (hunk 1 above)
Comment (beelchester) [reply .github/ISSUE_TEMPLATE/documentation.md]: Done 👍

**Comment by beelchester** [comment]
Hey @thisiskaransgit, I've made the changes that you have suggested can you please review it?

**Review by leecalcote — APPROVED** [review]
Thanks!

**Pull request merged**